crates/sequencer/src/commit.rs

```diff
@@ -0,0 +1,29 @@
+use blockifier::state::{
+    cached_state::CachedState,
+    state_api::{State as BlockifierState, StateReader as BlockifierStateReader},
+};
+
+pub trait Committer<S>
```

Comment:
is this naming taken from Blockfier?

Reply:
Blockifier uses `commit` and `stage` function names, so I figured Commiter, being an entity that can commit makes sense.